start.py

```diff
@@ -55,7 +56,17 @@ def main(simulation_parameters):
             allagents.do('filobl')
             insurancecustomers.do('check_risk')
 
-        simulation.graphs()
+        if not isTravis:
+            simulation.graphs()
```

Comment:
@rht I agree with @x0range this is not the proper way to handle this issue. It should be possible to run a simulation without a GUI using ABCE.  If it is not possible, then this is an issue for @DavoudTaghawiNejad to fix upstream in ABCE.  Until then we can focus on testing logic that does not require ABCE.  

Come to think of it how @DavoudTaghawiNejad mentioned that he has tests running on Travis, how is he solving this problem?